### ISSUE TITLE
Conditions for email notifications don't work properly with grouped dropdown fields

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#595]
**Description**
A client has been having trouble setting up a conditional email notification using a grouped dropdown field

When setting up the condition the fields values don't appear as they do with a regular dropdown

![Screen Shot 2021-10-07 at 3 51 58 pm](https://user-images.githubusercontent.com/21274925/136324634-97388248-2807-4e2f-90c3-49bb48e3a5dd.png)

![Screen Shot 2021-10-07 at 3 48 29 pm](https://user-images.githubusercontent.com/21274925/136324446-5d2204d7-8464-43bb-8594-4f1f62ca5dab.png)

We also noticed the logs have the following error when the form is submitted regardless if the condition is set up with a drop down or regular text type field. Though the condition seems to trigger properly for text fields 

`[-][1][-][error][formie] Failed to parse conditional “new910-5346containsTest”: “Argument 1 passed to craft\helpers\StringHelper::contains() must be of the type string, null given, called in /app/vendor/verbb/formie/src/helpers/ConditionsHelper.php on line 27” /app/vendor/craftcms/cms/src/helpers/StringHelper.php:272`

**Steps to reproduce**
1. Create a form with a dropdown field in a group
2. Try to set up an email notification condition using the fields value

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.19
- Craft version: 3.7.16
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Thanks for the quick fix!

The drop down field values are appearing correctly now, but unfortunately it looks like conditions still aren't triggering correctly for any grouped fields

If it helps, we still have the `Failed to parse conditional` error appearing in the formie logs when testing with a grouped text field using the `contains` condition, but not with a drop down

Have also noticed field show/hide conditions do work correctly for grouped fields

**Comment by engram-design** [comment]
Just pushed a fix for this, if you can fire the same command as the above and give that a go?

**Comment by d-karstens** [comment]
Yep, works perfect now! 👍 
Cheers for that

**Comment by engram-design** [comment]
Fixed in [1.4.20](https://github.com/verbb/formie/releases/tag/1.4.20)
